### ISSUE TITLE
(hardware) Keys not responding

### ISSUE THREAD

**Issue by submitaudio** [cfoulc/Number9#2]
Hi love the sampler!
but would love to use all the hardware keys.
the: e, y, u, i, a, keys are not working. and i have not used the keys for other shortcuts.
I use a Windows Surface Pro.

**Comment by cfoulc** [comment]
Hello, thanks for your interest and support.
please tell me your operating system so i can try to reproduce the problem.

**Comment by cfoulc** [comment]
if you have a way to test another keyboard, that would be interesting too.

until we found a solution, you can try to map midi keyboard keys to those
"screen" keys.

Le jeu. 28 nov. 2019 à 09:31, Triphase clément foulc <c.foulc@gmail.com> a
écrit :

**Comment by cfoulc** [comment]
been thinking, "e" "y" "u" "i" and "a" are letters that have modifications,
accents ("à","â","ª","æ", etc) so i guess your system is not transmitting
them the usual way, may be some system setting could allow you to use them
normally.

Le jeu. 28 nov. 2019 à 09:55, Triphase clément foulc <c.foulc@gmail.com> a
écrit :

**Comment by submitaudio** [comment]
Yes your where absolutely Wright, I am a Apple user and installed a special keyboard app to use special characters on the Windows laptop like é ü hehe uninstalled it and now it's working 👍🏼
